package.json

```diff
@@ -30,52 +30,45 @@
     "eslint-plugin-no-null": "^1.0.2",
     "eslint-plugin-no-unsanitized": "^3.1.4",
     "eslint-plugin-react": "^7.21.5",
-    "if-env": "^1.0.4",
     "ignore-styles": "^5.0.1",
     "jsdom": "^11.5.1",
-    "lerna": "^2.2.0",
+    "lerna": "^4.0.0",
     "nyc": "^15.0.0",
     "rimraf": "^2.6.1",
     "sinon": "^3.3.0",
     "temp": "^0.8.3",
     "tslint": "^5.12.0",
-    "typedoc": "^0.17.7",
+    "typedoc": "^0.21.2",
     "typedoc-plugin-external-module-map": "^1.2.1",
     "typescript": "^3.9.2",
     "uuid": "^8.0.0"
   },
   "scripts": {
     "preinstall": "node-gyp install",
     "postinstall": "node scripts/post-install.js",
-    "prepare": "yarn prepare:references && yarn --cwd packages/core prepare && yarn prepare:build && yarn prepare:hoisting && yarn download:plugins",
-    "prepare:references": "node scripts/compile-references.js",
-    "prepare:build": "yarn build && run lint && run build \"@theia/example-*\" --stream --parallel",
-    "prepare:hoisting": "theia check:hoisted -s",
-    "clean": "yarn lint:clean && node scripts/run-reverse-topo.js yarn clean",
-    "build": "tsc -b configs/root-compilation.tsconfig.json",
-    "watch": "tsc -b configs/root-compilation.tsconfig.json -w",
-    "lint": "run lint",
+    "prepare": "yarn tsreferences && lerna run prepare && yarn build:ts",
```

Comment:
I kind of regret that we don't have a single command anymore that will "build everyting" after checking out the source. As I understand it, we'll have to `yarn install && yarn build` now, right?

Reply:
I can add a `yarn everything` or something similiar, otherwise yes you have to do what you said.

Reply:
Added a `yarn all` command that does everything again.